### DISCUSSION TITLE
How can I enable openblas?

### DISCUSSION THREAD

**Discussion by newTomas** [Atome-FE/llama-node#65] [Q&A]
When using a long prompt, processing takes a very long time, openblas helps a lot and does not require a video card. But how to enable it in llama-node?

**Comment by hlhr202** [comment]
i m currently working on cublas. for openblas probably will try it later

**Reply by newTomas** [reply]
Are they hard to add at all? Maybe I can help?

**Reply by hlhr202** [reply]
you can try modifying build.rs for llama-sys, which uses cmake flags to build features for different architectures.

**Reply by hlhr202** [reply]
emm, you can use "feature/cuda" branch if you wish to pr for this. I v changed a lot build script there.
